server/sonar-server/src/main/java/org/sonar/server/app/WebServerProcessLogging.java

```diff
@@ -50,8 +48,5 @@ protected void extendLogLevelConfiguration(LogLevelConfig.Builder logLevelConfig
 
   @Override
   protected void extendConfigure() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)